src/react-components/libraryIndexView.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+import { useAppSelector, useAppDispatch } from '../redux/store';
+import { fetchLibraries } from '../redux/libraryIndex';
+
+export const LibraryIndexView = () => {
+  const libraryIndex = useAppSelector(state => state.libraryIndex);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(fetchLibraries())
+  }, [])
+
+  return (
+    <div>
+      <h2>List of Libraries</h2>
+      {libraryIndex.loading && <div>Loading...</div>}
+      {!libraryIndex.loading && libraryIndex.error ? <div>Error: {libraryIndex.error}</div> : null}
```

Comment:
You can technically use the same pattern as above here:
`{!libraryIndex.loading && libraryIndex.error && <div>Error: {libraryIndex.error}</div>}`
Nothing wrong with a ternary, just saw the difference between line 16 and this one.